src/app/pages/PerpetualPage/contexts/PerpetualQueriesContext.tsx

```diff
@@ -164,6 +177,26 @@ export const PerpetualQueriesContextProvider: React.FC<PerpetualQueriesContextPr
     [refetchContract, refetchUser],
   );
 
+  const refetchBalance = useMemo(
+    () =>
+      debounce(
+        () =>
+          bridgeNetwork
+            .call(
+              Chain.BSC,
+              getContract('PERPETUALS_token').address,
+              marginTokenAbi,
+              'balanceOf',
+              [account],
+            )
+            .then(result => result && setAvailableBalance(String(result)))
+            .catch(console.error),
+        500,
```

Comment:
Why do you have `500 ms` here? We use `1000 ms` (`THROTTLE_DELAY`) everywhere else.

Reply:
Idk sounded like a reasonable value :D